nix/haskell.nix

```
@@ -127,5 +133,8 @@ let
     ];
   };
 
+  # Hydra will pass the GitHub PR number as a string argument to release.nix.
+  isHydraPRJobset = toString pr != "";
```

```suggestion
  isHydraPRJobset = pr != "";
```
`toString` not necessary here.